Error looks related

```
Running "mocha:app_manager#fields" (mocha) task
Testing: http://localhost:8000/mocha/app_manager/fields
  DrilldownToFormController - Initialization
  1) Basic Form init
  2) Multiple App Types init
  3) Case Types init
  ✓ registers server error 
  ✓ registers HTTP error 
  2 passing (6ms)
  3 failing
  1) DrilldownToFormController - Initialization Basic Form init:
     AssertionError: expected [] to equal undefined
      at http://localhost:8000/static/chai/chai.js:210
      at assertEqual (http://localhost:8000/static/chai/chai.js:752)
      at http://localhost:8000/static/chai/chai.js:3854
      at http://localhost:8000/static/chai/chai.js:2507
      at http://localhost:8000/static/app_manager/spec/app_data_drilldown.DrilldownToFormController.init.spec.mo.js:49
  2) DrilldownToFormController - Initialization Multiple App Types init:
     AssertionError: expected [] to equal undefined
      at http://localhost:8000/static/chai/chai.js:210
      at assertEqual (http://localhost:8000/static/chai/chai.js:752)
      at http://localhost:8000/static/chai/chai.js:3854
      at http://localhost:8000/static/chai/chai.js:2507
      at http://localhost:8000/static/app_manager/spec/app_data_drilldown.DrilldownToFormController.init.spec.mo.js:98
  3) DrilldownToFormController - Initialization Case Types init:
     AssertionError: expected [ { id: 'case', text: 'Case' } ] to equal undefined
      at http://localhost:8000/static/chai/chai.js:210
      at assertEqual (http://localhost:8000/static/chai/chai.js:752)
      at http://localhost:8000/static/chai/chai.js:3854
      at http://localhost:8000/static/chai/chai.js:2507
      at http://localhost:8000/static/app_manager/spec/app_data_drilldown.DrilldownToFormController.init.spec.mo.js:142
>> 3/5 tests failed (0.01s)
Warning: Task "mocha:app_manager#fields" failed. Use --force to continue.
```

--- a/corehq/apps/app_manager/static/app_manager/spec/app_data_drilldown.DrilldownToFormController.init.spec.mo.js
+++ b/corehq/apps/app_manager/static/app_manager/spec/app_data_drilldown.DrilldownToFormController.init.spec.mo.js
@@ -46,7 +46,7 @@ describe("DrilldownToFormController - Initialization", function () {
             controller._select2Test.form.placeholder,
             response.placeholders.form
         );
-        assert.isUndefined(controller._select2Test.case_type.data);
+        assert.deepEqual(controller._select2Test.case_type.data, []);
         assert.equal(controller._select2Test.case_type.defaults, '');
         assert.isUndefined(controller._select2Test.case_type.placeholder);
     });
@@ -95,7 +95,7 @@ describe("DrilldownToFormController - Initialization", function () {
             controller._select2Test.form.placeholder,
             response.placeholders.form
         );
-        assert.isUndefined(controller._select2Test.case_type.data);
+        assert.deepEqual(controller._select2Test.case_type.data, []);
         assert.equal(controller._select2Test.case_type.defaults, '');
         assert.isUndefined(controller._select2Test.case_type.placeholder);
     });
@@ -139,7 +139,8 @@ describe("DrilldownToFormController - Initialization", function () {
         assert.isUndefined(controller._select2Test.form.data);
         assert.equal(controller._select2Test.module.defaults, '');
         assert.isUndefined(controller._select2Test.form.placeholder);
-        assert.isUndefined(controller._select2Test.case_type.data);
+        assert.deepEqual(controller._select2Test.case_type.data,
+                         [{id: 'case', text: "Case"}]);
         assert.equal(controller._select2Test.case_type.defaults, '');
         assert.equal(
             controller._select2Test.case_type.placeholder,

--- a/corehq/apps/export/forms.py
+++ b/corehq/apps/export/forms.py
@@ -175,7 +175,7 @@ class CreateExportTagForm(forms.Form):
                         'application',
                         placeholder=_("Select Application"),
                         ng_model="formData.application",
-                        ng_change="formData.model_type === updateModuleChoices()",
+                        ng_change="updateModuleChoices()",
                         ng_required="formData.model_type === 'form'",
                     ),
                     crispy.Field(

